share/spice/tor_node/tor_node.js

```diff
@@ -79,33 +150,47 @@
         }
 
         // Configure the template.
-        base.templates.group = "text";
-	base.templates.options.content = "record";
-	base.templates.options.keySpacing = true;
-        base.meta.sourceUrl = node.url;
+        base.templates = {
+            group: "text",
+            options: {
+                content: "record",
+                keySpacing: true
+            },
+            meta: {
+                sourceUrl: node.url
+            }
+        };
 
         return base;
     }
 
     function multipleNodes(base) {
         base.data = base.nodes;
-        base.templates.group = "icon";
-        base.templates.detail = false;
-        base.templates.item_detail = false;
-        base.templates.options.footer = Spice.tor_node.footer;
+        base.templates = {
+            group: "icon",
+            detail: false,
+            item_detail: false,
+            options: {
+                footer: Spice.tor_node.footer
+            },
+            variants: {
+                tileTitle: '1line-large',
+                tileSnippet: 'small'
+            }
+        };
 
         base.normalize = function(node) {
             // Handle the node's locale.
             node.icon = DDG.settings.region.getSmallIconURL(getCountryCode(node.country));
 
             // Handle the node's uptime history.
-            node.uptime = timeFromNow(node.last_restarted);
-            node.downtime = timeFromNow(node.last_seen);
+            node.uptime = moment(node.last_restarted).fromNow(true);
```

Comment:
One last suggestion would be to use `sort_fields` and `sort_default:` to sort the tiles by uptime. I've experimented with this, it looks like you'd need to create a new date object maybe in Unix timestamp format then compare the two within `sort_fields`